server/test/service/ScalaTestDynamicRunnerTest.scala

```diff
@@ -1,8 +1,9 @@
 package service
 
-import org.scalatest.FlatSpecLike
+import org.scalatest._
+import service.ScalaTestRunnerTest._
 
-class ScalaTestDynamicRunnerTest extends ScalaTestRunnerTest with FlatSpecLike {
```

Comment:
Not really. getReport was override and Dynamic test is supposed to have 5 test case. That means Dynamic test using the same test cases as static runner and this prove that both Dynamic and Static Runners are working equally. Now, having no confirmation that Dynamic is not falling on the same test cases as for Static Runner we lost ability to check it without such test. So test should be restored.